cli/src/commands/working-groups/verifyValidator.ts

```
@@ -0,0 +1,60 @@
+import { flags } from '@oclif/command'
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+import { VerifyValidator, RemarkMetadataAction } from '@joystream/metadata-protobuf'
+import { metadataToString } from '../../helpers/serialization'
+import Long from 'long'
+import chalk from 'chalk'
+
+export default class VerifyValidatorCommand extends WorkingGroupsCommandBase {
+  static description =
+    'Verify or un-verify the membership profile bound to a validator account. Available to membership workers only.'
+
+  static args = [
+    {
+      name: 'memberId',
+      required: true,
+      description: 'ID of the membership bound to the validator account.',
+    },
+  ]
+
+  static flags = {
+    unverify: flags.boolean({
+      default: false,
+      description: 'Whether the profile should be un-verified.',
+    }),
+
+    ...WorkingGroupsCommandBase.flags,
+  }
+
+  async init(): Promise<void> {
+    await super.init()
+    this._group = WorkingGroups.Membership
+  }
```

```
 ./bin/run working-groups:verifyValidator 2 
Initializing the query node connection (http://localhost:8081/graphql)...... done
Initializing the api connection (ws://localhost:9944)...... done
Current Group: storageProviders
```
Due to this change, whenever you run the command `Current Group: storageProviders` [log](https://github.com/thesan/joystream/blob/6ffd6ec85e03ebf3e19f95f8c683c133f062c103/cli/src/base/WorkingGroupsCommandBase.ts#L88) will be printed, which might be confusing for the user

Also, instead of changing the group in code wouldn't it better to throw the error if user didn't specify the `--group=membership` flag (i.e. explicitly asking user to run command in membership group context)